src/main/java/Omoh.java

```diff
@@ -16,10 +16,8 @@ public static void main(String[] args) {
     }
 
     //Method that prints horizontal line using "_" char
-    private static void printHorizontalLine() {
-        for (int j = 0; j < 4; j++) {
-            System.out.print(" ");
-        }
+    public static void printHorizontalLine() {
+        System.out.print("    ");
```

Comment:
Great job in including spaces to make code more user friendly. Could avoid DRY rule in by possibly including it in your class as a part of a method